plugins/inputs/prometheus/README.md

```diff
@@ -19,30 +19,37 @@ in Prometheus format.
   ##   example: metric_version = 1; 
   ##            metric_version = 2; recommended version
   # metric_version = 1
-
+  
+  ## Url tag name (tag containing scrapped url. optional, default is "url")
+  # url_tag = "scrapeUrl"
```

Comment:
The value should represent the default
```suggestion
  # url_tag = "url"
```

Reply:
if we change it here, we need to change it here as well: https://github.com/influxdata/telegraf/blob/master/plugins/inputs/prometheus/prometheus.go#L94

Reply:
Correct!